### PR TITLE
motionEye: add SNTP support

### PR DESCRIPTION
**Pull request by ccrisan**

### CODE REVIEW

File: package/motioneye/extractl.py [hunk 1]
```diff
@@ -449,7 +449,7 @@ def dateInterval():
         'section': 'expertSettings',
         'reboot': True,
         'required': True,
-        'depends': ['dateMethod==http'],
+        'depends': ['dateMethod==(http'],
```
Comment (popoviciri) [inline package/motioneye/extractl.py]: This should read `'depends': ['dateMethod==(http|sntp)'],`

**Comment by popoviciri** [comment]
The dateMethod is being correctly assigned in `date.conf`.
But while still working on this, perhaps another bug can be fixed. Here is the content of the `sntp.log` file:
- during boot:
```
sntp 4.2.8p13@1.3847-o Mon Oct  7 23:01:17 UTC 2019 (1)
1970-01-01 01:00:16.049563 (-0100) +1570528554.31769 +/- 1047019036.711902 10.145.180.1 s1 no-leap
```
It appears that the sntp runs before the network is up and running. Although the date is reported correctly as soon as I can ssh into the pi.
- after running `/etc/init.d/*date start` the log file looks like this:
```
sntp 4.2.8p13@1.3847-o Mon Oct  7 23:01:17 UTC 2019 (1)
2019-10-08 12:12:46.574042 (-0100) +0.00114 +/- 0.500942 10.145.180.1 s1 no-leap
```
At last, I was expecting that after the `dateInterval` time the `sntp.log` file will get an update. This did not happen.
Perhaps all this is expected. Just wanted to share my test results :).
Cheers!

File: package/motioneye/extractl.py (hunk 1 above)
Comment (ccrisan) [reply package/motioneye/extractl.py]: Oups, I have no idea how this has slipped here. In any case, the date interval property is only used with HTTP method. SNTP is only used at startup, while NTP is responsible for maintaining the system date thereafter.

**Comment by ccrisan** [comment]
@popoviciri regarding the wrong date at boot, that's very strange since network is definitely initialized by that time. Is this 100% reproducible?

> At last, I was expecting that after the `dateInterval` time the `sntp.log` file will get an update. This did not happen.

SNTP is only used at boot for an initial time update. NTP is responsible thereafter.

**Comment by ccrisan** [comment]
@popoviciri in the meantime I'm merging this PR since it's not its fault if SNTP still fails. We'll have to track that somewhere else.

**Pull request merged**